pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -4,10 +4,20 @@
 , lib
 }:
 
-args@{ name, bazelFlags ? [], bazelBuildFlags ? [], bazelFetchFlags ? [], bazelTarget, buildAttrs, fetchAttrs, ... }:
+args@{
+  name
+, bazelFlags ? []
+, bazelBuildFlags ? []
+, bazelFetchFlags ? []
+, bazelTarget
+, buildAttrs
+, fetchAttrs
+, removeRulesCC ? true
```

Comment:
Can you add the description from the PR as a comment here?

Reply:
@Profpatsch - done!